contracts/Users.sol

```diff
@@ -118,6 +122,17 @@ contract Users {
         return coll;
     }
 
+    function setUsers(uint newLimit) onlyOwner returns (bool) {
```

Comment:
`setUsers()` is confusing I guess. You should use another name like `setMaxLimit`.

Reply:
ok i just added from my side but yeah you are right on that